weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallDetails.tsx

```diff
@@ -53,9 +50,26 @@ export const CallSchemaLink = ({call}: {call: CallSchema}) => {
   );
 };
 
+const useParentCall = (
+  call: CallSchema | null
+): Loadable<CallSchema | null> => {
+  const {useCall} = useWFHooks();
+  let parentCall = null;
+  if (call && call.parentId) {
+    parentCall = {
+      entity: call.entity,
+      project: call.project,
+      callId: call.parentId,
+    };
+  }
+  return useCall(parentCall);
+};
+
 export const CallDetails: FC<{
   call: CallSchema;
 }> = ({call}) => {
+  const {useCalls} = useWFHooks();
```

Comment:
You will see this pattern in many places now - you must get the hooks from a hook since they will change based on the project's backing data store.